### PR TITLE
Kanidm book review 6

### PR DESCRIPTION
**Pull request by cjschroder**
Fixes #

- [ ] cargo fmt has been run
- [ ] cargo clippy has been run
- [ ] cargo test has been run and passes
- [ ] book chapter included (if relevant)
- [ ] design document included (if relevant)

### CODE REVIEW

File: kanidm_book/src/pam_and_nsswitch.md [hunk 1]
```diff
@@ -263,11 +281,18 @@ edit the content.
 
 #### With authselect
 To work with authselect:
-You will need to [create a new profile](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/configuring_authentication_and_authorization_in_rhel/configuring-user-authentication-using-authselect_configuring-authentication-and-authorization-in-rhel#creating-and-deploying-your-own-authselect-profile_configuring-user-authentication-using-authselect). First run
+
+You will need to 
+[create a new profile](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/configuring_authentication_and_authorization_in_rhel/configuring-user-authentication-using-authselect_configuring-authentication-and-authorization-in-rhel#creating-and-deploying-your-own-authselect-profile_configuring-user-authentication-using-authselect).
```
Comment (yaleman) [inline kanidm_book/src/pam_and_nsswitch.md:286]: Maybe this similar length but more relevant one?

https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/configuring_authentication_and_authorization_in_rhel/configuring-user-authentication-using-authselect_configuring-authentication-and-authorization-in-rhel#creating-and-deploying-your-own-authselect-profile_configuring-user-authentication-using-authselect

File: kanidm_book/src/radius.md [hunk 2]
```diff
@@ -165,25 +168,26 @@ You can then run the container with:
 
     docker run --name radiusd -v ...:/data kanidm/radius:latest
 
-Authentication can be tested through the client.localhost NAS configuration with:
+Authentication can be tested through the client.localhost Network Attached Storage (NAS) configuration with:
```
Comment (yaleman) [inline kanidm_book/src/radius.md:171]: It's actually Network Access Server (NAS), because yay namespace collisions!

**Review by yaleman — CHANGES_REQUESTED** [review]
Two comments, thanks for the review 😄

File: kanidm_book/src/radius.md (hunk 2 above)
Comment (Firstyear) [reply kanidm_book/src/radius.md:171]: NAS (Lil X).

File: kanidm_book/src/pam_and_nsswitch.md (hunk 1 above)
Comment (cjschroder) [reply kanidm_book/src/pam_and_nsswitch.md:286]: That is the same link.

File: kanidm_book/src/radius.md (hunk 2 above)
Comment (cjschroder) [reply kanidm_book/src/radius.md:171]: Thanks :) Network Attached Storage didn't make sense, but who knows with these newfangled services.

**Review by Firstyear — APPROVED** [review]

**Comment by Firstyear** [comment]
@yaleman I'm good with this!

File: kanidm_book/src/pam_and_nsswitch.md (hunk 1 above)
Comment (Firstyear) [reply kanidm_book/src/pam_and_nsswitch.md:286]: derp hard or herp home, I always say.

Thanks as always, you are both fantastic!

**Pull request merged**